### PR TITLE
Tooling general guide

### PR DESCRIPTION
**Pull request by Amuhar**

### CODE REVIEW

File: docs/guides/tooling.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Validator Ejector
+
+Daemon service which loads LidoOracle events for validator exits and sends out exit messages when necessary.
+
+- **Version**: 1.2.0
+- **Docker image**: sha256:e3acb5eb1bab5a871bf70e9bb736e1a1016567e1846a9b96c27232c84643597a, [lidofinance/validator-ejector@sha256-e3acb5eb1bab5a871bf70e9bb736e1a1016567e1846a9b96c27232c84643597a](https://hub.docker.com/layers/lidofinance/validator-ejector/1.2.0/images/sha256-e3acb5eb1bab5a871bf70e9bb736e1a1016567e1846a9b96c27232c84643597a?context=explore)
+- **Commit hash**: [lidofinance/validator-ejector@c2d05cb](https://github.com/lidofinance/validator-ejector/commit/c2d05cbfff039a8332f4ae5994fc1148b8cbf154)
+- **Last update date**: 26 April, 2023
+- [**Repository**](https://github.com/lidofinance/validator-ejector/tree/1.2.0#readme)
+- [**Documentation**](https://docs.lido.fi/guides/validator-ejector-guide)
```
Comment (avsetsin) [inline docs/guides/tooling.md:25]:
```suggestion
- [**Documentation**](/guides/validator-ejector-guide)
```

File: docs/guides/tooling.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Council daemon
+
+The Lido Council Daemon monitors deposit contract keys.
+
+- **Version**: 1.8.2
+- **Docker image**: sha256:db99dd3494301abd586a77692154a6eef62b6e07ef3e60e49a0e6f0f5495865a, [lidofinance/lido-keys-api@sha256-db99dd3494301abd586a77692154a6eef62b6e07ef3e60e49a0e6f0f5495865a](https://hub.docker.com/layers/lidofinance/lido-keys-api/0.8.0/images/sha256-db99dd3494301abd586a77692154a6eef62b6e07ef3e60e49a0e6f0f5495865a?context=explore)
+- Commit hash: [lidofinance/lido-council-daemon@e10a1dc](https://github.com/lidofinance/lido-council-daemon/commit/e10a1dcce34b118e0e830efafa44198f4e865414)
+- **Last update date**: 27 April, 2023
+- [**Repository**](https://github.com/lidofinance/lido-council-daemon/tree/1.8.2)
```
Comment (avsetsin) [inline docs/guides/tooling.md:35]:
```suggestion
- [**Repository**](https://github.com/lidofinance/lido-council-daemon/tree/1.8.2)
- [**Documentation**](/guides/deposit-security-manual)
```

File: docs/guides/tooling.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- **Last update date**: 26 April, 2023
+- [**Repository**](https://github.com/lidofinance/validator-ejector/tree/1.2.0#readme)
+- [**Documentation**](https://docs.lido.fi/guides/validator-ejector-guide)
+
+## Council daemon
+
+The Lido Council Daemon monitors deposit contract keys.
+
+- **Version**: 1.8.2
+- **Docker image**: sha256:db99dd3494301abd586a77692154a6eef62b6e07ef3e60e49a0e6f0f5495865a, [lidofinance/lido-keys-api@sha256-db99dd3494301abd586a77692154a6eef62b6e07ef3e60e49a0e6f0f5495865a](https://hub.docker.com/layers/lidofinance/lido-keys-api/0.8.0/images/sha256-db99dd3494301abd586a77692154a6eef62b6e07ef3e60e49a0e6f0f5495865a?context=explore)
```
Comment (avsetsin) [inline docs/guides/tooling.md:32]:
```suggestion
- **Docker image**: sha256:42ece4c9484aac0ee192a72407f149cb062305596c0c6b5bb0a91a7b8f4bb3db, [lidofinance/lido-council-daemon@sha256-42ece4c9484aac0ee192a72407f149cb062305596c0c6b5bb0a91a7b8f4bb3db](https://hub.docker.com/layers/lidofinance/lido-council-daemon/1.8.2/images/sha256-42ece4c9484aac0ee192a72407f149cb062305596c0c6b5bb0a91a7b8f4bb3db?context=explore)
```

File: docs/guides/tooling.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+## Keys API
+
+Lido keys HTTP API.
+
+- **Version**: 0.8.1
+- **Docker image**: sha256:76f5a48f0baab46ada3bf4b09bdb115a74803a7ba53e783e505436d8177383dc, [lidofinance/lido-keys-api@sha256-76f5a48f0baab46ada3bf4b09bdb115a74803a7ba53e783e505436d8177383dc](https://hub.docker.com/layers/lidofinance/lido-keys-api/0.8.1/images/sha256-76f5a48f0baab46ada3bf4b09bdb115a74803a7ba53e783e505436d8177383dc?context=explore)
+- **Commit hash**: [lidofinance/lido-keys-api@d0e4080](https://github.com/lidofinance/lido-keys-api/commit/d0e4080a8982dca4aed97aa8f7f76ab7411a665d)
+- **Last update date**: 3 May, 2023
+- [**Repository**](https://github.com/lidofinance/lido-keys-api/tree/0.8.1#readme)
+- [**Documentation**](https://docs.lido.fi/guides/kapi-guide)
```
Comment (avsetsin) [inline docs/guides/tooling.md:46]:
```suggestion
- [**Documentation**](/guides/kapi-guide)
```

**Review by avsetsin — APPROVED** [review]
lgtm, but KAPI and Ejector will require detailed pages

**Review by TheDZhon — APPROVED** [review]
Great start :+1:

**Review by eddort — APPROVED** [review]

**Pull request merged**